RecoTracker/LSTCore/src/alpaka/TrackCandidate.h

```diff
@@ -223,10 +243,22 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
             float phi2 = __H2F(quintuplets.phi()[quintupletIndex]);
             float dEta = alpaka::math::abs(acc, eta1 - eta2);
             float dPhi = cms::alpakatools::deltaPhi(acc, phi1, phi2);
-
             float dR2 = dEta * dEta + dPhi * dPhi;
-            if (dR2 < 1e-3f)
-              pixelSegments.isDup()[pixelArrayIndex] = true;
+            // Cut on pLS-T5 embed distance.
+            if (dR2 < 0.02f) {
+              float d2 = 0.f;
+              CMS_UNROLL_LOOP for (unsigned k = 0; k < Params_pLS::kEmbed; ++k) {
+                float diff = pixelSegments.plsEmbed()[pixelArrayIndex][k] - quintuplets.t5Embed()[quintupletIndex][k];
```

Comment:
`pixelSegments.plsEmbed` temporary could be outside the loop, although it's probably going to stay in cache anyways and have no diff in performance